### PR TITLE
Reset design outlet humidity ratio when entering humidity ratio is less to prevent SHR>1 in DX coil sizing

### PR DESCRIPTION
**Pull request by Nigusse**
Pull request for 90946736_SHRgt1_4567.  
Addresses #4893 

This fix traps and resets bad inputs that may results in SHR greater than 1 for DX and chilled water cooling coils. If the design supply air humidity ratio is greater than the cooling coil entering air (mixed air or return air) humidity ratio, the EnergyPlus sizing calculation internally resets it to entering condition to avoid SHR greater than 1.0 situation.

### CODE REVIEW

**Comment by mjwitte** [comment]
@Myoldmopar Looks like ubuntu CI is stuck?  This build has been pending since yesterday afternoon.

**Comment by Myoldmopar** [comment]
Appears to be fine, @mjwitte.  They just got a little clogged yesterday afternoon.

File: tst/EnergyPlus/unit/ReportSizingManager.unit.cc [hunk 1]
```diff
@@ -124,3 +140,169 @@ TEST( ReportSizingManager, GetCoilDesFlowT )
...
+
+	std::string CompName; // component name
+	std::string CompType; // component type
+	std::string SizingString; // input field sizing description
+	int SizingType; // integer type of sizing requested
+	Real64 SizingResult; // autosized value of coil input field
+	bool PrintWarning; // true when sizing information is reported in the eio file
+	std::string CallingRoutine; // calling routine
+
+	DataHVACGlobals::CoolingCapacitySizing;
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/ReportSizingManager.unit.cc]: @Nigusse Just noticed some of the CI builds produce warnings about this line.  I'm assuming the intent here was to set this to true?  If so, I can make the change.

File: tst/EnergyPlus/unit/ReportSizingManager.unit.cc [hunk 2]
```diff
@@ -124,3 +140,169 @@ TEST( ReportSizingManager, GetCoilDesFlowT )
...
+	int const ZoneNum = 1;
+	std::string CompName; // component name
+	std::string CompType; // component type
+	std::string SizingString; // input field sizing description
+	int SizingType; // integerized type of sizing requested
+	Real64 SizingResult; // autosized value of coil input field
+	bool PrintWarning; // true when sizing information is reported in the eio file
+	std::string CallingRoutine; // calling routine
+
+	DataHVACGlobals::CoolingCapacitySizing;
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/ReportSizingManager.unit.cc]: @Nigusse Same here as well - supposed to be set to true?

**Comment by mjwitte** [comment]
@Myoldmopar Lots of failures [here](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-4fa3acafa13682a0b031fe6d5891d16af998578a-x86_64-Linux-Ubuntu-14.04-gcc-4.8.html) with server access problems.  I need to fix a couple lines here anyway, so these results will be overwritten, but wondering if we can do anything to do a retry when this happens?

File: tst/EnergyPlus/unit/ReportSizingManager.unit.cc (hunk 1 above)
Comment (mjwitte) [inline tst/EnergyPlus/unit/ReportSizingManager.unit.cc]: @Nigusse Never mind, these lines are just surplus.  SizingType is set to this elsewhere.

**Pull request merged**